jenkins/run_containers_pod.yml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: cloud-governance-pod-1
```

Comment:
cloud-governance-pod-1 - why we need -1 ?

Reply:
Ohh, sorry. Forgot to delete 1 after testing